test/account.go

```diff
@@ -88,6 +88,16 @@ var TestTenantIdentity = account.Identity{
 	User:     TestUser,
 }
 
+// CreateLonelyTestIdentity creates an identity not assosiated with any user. For testing purpose only.
+func CreateLonelyTestIdentity(db *gorm.DB, username string) (account.Identity, error) {
```

Comment:
The other methods used to created 'lonely' identities until I 'coupled' them off in one of my PRs :D